client/src/bastionlab/polars/utils.py

```diff
@@ -23,6 +34,54 @@ def create_byte_chunk(data: bytes) -> Tuple[int, Iterator[bytes]]:
 def serialize_dataframe(
     df: pl.DataFrame, policy: Policy, sanitized_columns: List[str]
 ) -> Iterator[SendChunk]:
+    #:  Converts Polars `DataFrame` to BastionLab `SendChunk`.
+    #:  Receives `polars.internals.dataframe.frame.DataFrame` and uses `__getstate__` to convert `DataFrame`
+    #:  to `List[bytes]`.
+
+    #: `__getstate__()` internally calls `Series.__getstate__()` which converts
+    #: Polars `Series` to `bytes`, and simply adds `b"[END]"` to the end of each `Series.__getstate__()`.
+
+    #: Parameters
+    #: ----------
+    #: df : polars.internals.dataframe.frame.DataFrame
+    #:     Polars DataFrame
+    #: policy : Policy
+    #:     BastionLab Remote DataFrame policy. This specifies which operations can be performed on
+    #:     DataFrames and they specified the _data owner_.
+    #: sanitized_columns : List[str]
+    #:     This field contains (sensitive) columns in the DataFrame that are to be removed when a Data Scientist
+    #:     wishes to fetch a query performed on the DataFrame.
+
+    #: Returns
+    #: -------
+    #: Iterator[SendChunk]
+
+    #: Example
+    #: -------
```

Comment:
To be frank, I am confused, why there are example for some examples only... Is there a reason for that?

Reply:
Yes, because this is a very important aspect of BastionLab and although an internal function, it's good to show how it works with an example explaining how it works.
The inspiration was gotten from Polars repo itself.

Reply:
If you start to do it somewhere, it is important to do it in more place as well, like what we can find in other Python or Rust lib.

Reply:
Yeah, gradually, everywhere would have it.
For the mean time, we have it in utils.
Especially for `remote_polars`, there will be lots.